scripts/deploy-to-cloud-run.sh

```
@@ -0,0 +1,136 @@
+set -e
+
+# Set project ID
+PROJECT_ID="fifiai"
+REGION="asia-southeast2"
+SERVICE_NAME="geo-tracking-service"
+TIMESTAMP=$(date +%Y%m%d-%H%M%S)
+
+# Load environment variables from .env file
+echo "Loading environment variables from .env file..."
+if [ -f ".env" ]; then
+  export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)